magento/pwa-studio
# Cypress updates

## Description
  - Update cypress dependencies 
  - Update test titles
  - Update consolidated result output to have title mapped  (mochawesome has bug  which creates file itself with wrong file path so switched to test title)
  - Make test compatible with 2.4.4
  - Update PB breaking snaps

## Related Issue







Closes https://jira.corp.magento.com/browse/PWA-2553

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






 - Run cypress suite locally with --t options and verify its pass.
 - Observer test results and make sure no duplication in test title


## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
